Source/WebKit/UIProcess/RemoteLayerTree/ios/RemoteLayerTreeDrawingAreaProxyIOS.mm

```diff
@@ -211,8 +212,11 @@ - (void)didChangeNominalFramesPerSecond
     return m_displayLinkHandler.get();
 }
 
-void RemoteLayerTreeDrawingAreaProxyIOS::setPreferredFramesPerSecond(FramesPerSecond preferredFramesPerSecond)
+void RemoteLayerTreeDrawingAreaProxyIOS::setPreferredFramesPerSecond(IPC::Connection& connection, FramesPerSecond preferredFramesPerSecond)
 {
+    if (!m_webProcessProxy->hasConnection() || &m_webProcessProxy->connection() != &connection)
```

Comment:
AuxiliaryProcessProxy::hasConnection(const IPC::Connection&) does this in one call.
Also, aren't we working towards removing m_webProcessProxy?